tests/spicy/tools/preprocessor.spicy

```diff
@@ -0,0 +1,60 @@
+# @TEST-EXEC: echo == spicyc >>output
```

Comment:
Since we support stray spaces in some places,  let's maybe make this test less "pretty", e.g.,

```.patch
diff --git a/tests/spicy/tools/preprocessor.spicy b/tests/spicy/tools/preprocessor.spicy
index 9df8b5d7..e69c043b 100644
--- a/tests/spicy/tools/preprocessor.spicy
+++ b/tests/spicy/tools/preprocessor.spicy
@@ -11,7 +11,7 @@

 module Foo;

-@if HAVE_ZEEK
+@if  HAVE_ZEEK
 print "have zeek";
 @else
 print "not have zeek";
@@ -25,11 +25,11 @@ print "not have zeek";
 print "have spicy version";
 @endif

-@if SPICY_VERSION >= 400
+@if SPICY_VERSION  >= 400
 print "have spicy version >= 0.4";
 @endif

-@if SPICY_VERSION >= 40000
+@if SPICY_VERSION >=  40000
 print "have spicy version >= 4";
 @endif

@@ -42,7 +42,7 @@ no valid Spicy syntax here.
 @endif

 @if HAVE_ZEEK
-    @if ZEEK_VERSION > 10000
+    @if ZEEK_VERSION  >  10000
     print "have zeek and zeek version > 1.0";
     @else
     print "have zeek but zeek version < 1.0";
```